classes/controller/AdminController.php

```diff
@@ -2145,7 +2150,12 @@ protected function addPageHeaderToolBarModulesListButton()
     {
         $this->filterTabModuleList();
 
-        if (is_array($this->tab_modules_list['slider_list']) && count($this->tab_modules_list['slider_list'])) {
+        /**
+         * @TODO remove isInstalled when MBO module will be released
```

Comment:
from my experience, this todo will be there in 3 years. Don't introduce it please :'(